audio/ffmpeg.py

```diff
@@ -124,11 +125,11 @@ def tasks(self):
 
     def run(self):
         c = corpus.Corpus()
-        c.load(tk.uncached_path(self.corpus_file))
+        c.load(self.corpus_file.get_path())
 
         from multiprocessing import pool
 
-        p = pool.Pool(self.rqmt["cpu"])
+        p = pool.Pool(self.rqmt["cpu"] // 2)
```

Comment:
should we remove 3 here for python, input and output?

Reply:
Why remove 3? You can remove 1 for Python, but doing the //2 for odds already does that.

Reply:
from your comment:

> e.g. 1 core for python and 4x2 cores for ffmpeg, one for input processing and one for output processing

oh.. sorry, I am bad at math this morning.. I thought you meant `1 + 4x2 + 1 + 1` here but now I realize that the result is 9 which is `1 + 4x(1+1)` (And I was searching for the extra two threads for "input and output processing").

Ok, then disregard this comment.